### DISCUSSION TITLE
Get minidump from crash

### DISCUSSION THREAD

**Discussion by kristjanvalur** [getsentry/sentry-native#1018] [Q&A] [answered]
Hi, we have been using this library for our game.  We are currently using the "breakpad" backend on windows and we do get crashes, but it would be useful, for the context of debug, to be able to access the associated minidumps.  Is this possible at all?  Are we missing something?

**Comment by supervacuus** [comment, marked as answer]
You can get the associated minidumps if you enable them in the WebUI via:

Project Settings -> Processing -> Security & Privacy -> Store Native Crash Reports

"Unlimited" means that any event will provide you with a minidump in the attachment section. You can download it and load it into WinDbg or the VS Debugger.

Also have a look at the docs here: https://docs.sentry.io/platforms/native/enriching-events/attachments/#enabling-crash-report-storage
